django/core/serializers/xml_serializer.py

```diff
@@ -53,7 +53,10 @@ def start_object(self, obj):
 
         self.indent(1)
         attrs = {"model": str(obj._meta)}
-        if not self.use_natural_primary_keys or not hasattr(obj, "natural_key"):
+
+        pk_included = self._should_include_pk(obj)
+
+        if pk_included:
```

Comment:
This method name is named well enough to just avoid the assignment:
```suggestion
        if self._should_include_pk(obj):
```